<ui version="4.0" >
 <class>TableViewWindow</class>
 <widget class="QDialog" name="TableViewWindow" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QTLs / Individuals view</string>
  </property>
  <layout class="QVBoxLayout" name="TableViewVLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tableViewTabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="qtlViewTab" >
      <attribute name="title" >
       <string>QTL columns view</string>
      </attribute>
      <layout class="QVBoxLayout" name="qtlViewTabVLayout" >
       <property name="margin" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="qtlViewIntroLabel" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>19</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>19</height>
          </size>
         </property>
         <property name="text" >
          <string>Use this tab to select wich QTL column must be hidden or shown.</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QPushButton" name="qtlAddPushButton" >
           <property name="minimumSize" >
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="text" >
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qtlRemovePushButton" >
           <property name="minimumSize" >
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="text" >
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qtlAllPushButton" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>27</height>
            </size>
           </property>
           <property name="text" >
            <string>All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qtlNonePushButton" >
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>27</height>
            </size>
           </property>
           <property name="text" >
            <string>None</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="qtlViewHorizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>6</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="qtlTableWidget" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="indivViewTab" >
      <attribute name="title" >
       <string>Individuals rows view</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <property name="margin" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="indivViewIntroLabel" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>19</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>19</height>
          </size>
         </property>
         <property name="text" >
          <string>Use this tab to select wich individual row must be hidden or shown.</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="indivAddPushButton" >
           <property name="minimumSize" >
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="text" >
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="indivRemovePushButton" >
           <property name="minimumSize" >
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="text" >
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="indivAllPushButton" >
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>27</height>
            </size>
           </property>
           <property name="text" >
            <string>All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="indivNonePushButton" >
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>27</height>
            </size>
           </property>
           <property name="text" >
            <string>None</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>6</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="switchLabel" >
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text" >
            <string>Choose cycle or group:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="switchComboBox" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>27</height>
            </size>
           </property>
           <item>
            <property name="text" >
             <string>Cycle</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Group</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="cycleComboBox" >
             <property name="minimumSize" >
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>16777215</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="groupComboBox" >
             <property name="minimumSize" >
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>16777215</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="indivHorizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="indivTableWidget" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="tableViewButtonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="configdialog.qrc" />
 </resources>
 <connections/>
</ui>
